docs/02-app/02-api-reference/04-functions/server-actions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Server Actions
+nav_title: Server Actions
+description: API Reference for Next.js Server Actions.
+related:
+  title: Next Steps
+  description: For more information on what to do next, we recommend the following sections
+  links:
+    - app/building-your-application/data-fetching/forms-and-mutations
+---
+
+{/* TODO: This page will need to link back to React docs mentioned at the bottom */}
+
+Next.js integrates with React Actions to provide a built-in solution for [server mutations](/docs/app/building-your-application/data-fetching/forms-and-mutations).
+
+## Convention
+
+You can enable Server Actions in your Next.js project by enabling the **experimental** `serverActions` flag.
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    serverActions: true,
+  },
+}
+```
+
+Server Actions can be defined in two places:
+
+- Inside the component that uses it (Server Components only).
+- In a separate file (Client and Server Components), for reusability. You can define multiple Server Actions in a single file.
+
+### With Server Components
+
+Create a Server Action by defining an asynchronous function with the [`"use server"`](https://react.dev/reference/react/use-server) directive at the top of the function body. `"use server"` ensures this function is only ever executed on the server.
+
+This function should have [serializable arguments](https://developer.mozilla.org/en-US/docs/Glossary/Serialization) and a [serializable return value](https://developer.mozilla.org/en-US/docs/Glossary/Serialization).
```

Comment:
Should these both be pointing to the same link? If so wondering if it's worth combining into one link

Reply:
Yeah! I wanted to avoid the super-long-string-of-blue.